packages/app/src/app/pages/Dashboard/Header/index.tsx

```diff
@@ -70,7 +70,7 @@ export const Header: React.FC<HeaderProps> = React.memo(
         </Link>
         <IconButton
           name="menu"
```

Comment:
On hover, the icon is misaligned to the bottom of the button.

![dgwcw8-3000 preview csb app_dashboard_settings (1)](https://user-images.githubusercontent.com/24959348/190205496-d6c78f17-6496-4844-9ecc-11bde8ca5c22.png)